includes/class-wc-stripe-api.php

```diff
@@ -16,6 +16,27 @@ class WC_Stripe_API {
 	const ENDPOINT           = 'https://api.stripe.com/v1/';
 	const STRIPE_API_VERSION = '2024-06-20';
 
+	/**
+	 * The Invalid API Keys cache key.
+	 *
+	 * @var string
+	 */
+	const INVALID_API_KEYS_CACHE_KEY = 'invalid_api_keys_rate_limit';
+
+	/**
+	 * The Invalid API Keys consecutive 401 error count threshold.
+	 *
+	 * @var int
+	 */
+	const INVALID_API_KEYS_COUNT_THRESHOLD = 5;
+
+	/**
+	 * The Invalid API Keys delay in seconds enforced for Stripe API calls after the threshold is reached.
+	 *
+	 * @var int
+	 */
+	const INVALID_API_KEYS_DELAY_IN_SECONDS = 2 * HOUR_IN_SECONDS;
```

Comment:
I used the same expiration as the account cache... we don't expect invalid keys to become valid again; this is more of a false positive prevention fallback (discussed for 9.5.1)

Reply:
That's fair. I would also think it may be worth explicitly calling this a `CACHE_TIMEOUT` rather than a `DELAY`.
```suggestion
	const INVALID_API_KEYS_CACHE_TIMEOUT_IN_SECONDS = 2 * HOUR_IN_SECONDS;
```